hw06/src/main/java/org/otus/hw06/storage/Storage.java

```diff
@@ -0,0 +1,16 @@
+package org.otus.hw06.storage;
+
+import org.otus.hw06.banknote.Banknote;
+import org.otus.hw06.banknote.Nominal;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public interface Storage {
+    public void setBanknote(Banknote banknote);
+    public List<Banknote> getBanknote(Map<Nominal, Integer> map);
+    public int getBalance();
+    public Map<Nominal, Integer> getNominal();
```

Comment:
```suggestion
    public Map<Nominal, Integer> getCells();
```

Есть перечисление Nominal и по названию метода возвращается именно это перечисление, но это не так.